app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Good job working on this but please kindly ensure this model belongs to author just like in the Post model.